lld/test/ELF/icf-preemptible.s

```diff
@@ -17,7 +17,9 @@
 # EXE-NEXT: selected section {{.*}}:(.text.h1)
 # EXE-NEXT:   removing identical section {{.*}}:(.text.h2)
 # EXE-NEXT:   removing identical section {{.*}}:(.text.h3)
-# EXE-NOT:  {{.}}
+# EXE-NEXT: Redirecting f2 to f1
+# EXE-NEXT: Redirecting g2 to g1
+# EXE-NEXT: Redirecting g3 to g1
```

Comment:
Could add `EXE-NOT: {{.}}` to ensure that there is no new line below (to catch bugs like printing `redirecting` multiple times)